cgsmiles/resolve.py

```diff
@@ -177,9 +178,12 @@ def edges_from_bonding_descrpt(self):
 
             # bonding descriptors are assumed to have bonding order 1
             # unless they are specifically annotated
-            order = re.findall("\d+\.\d+", bonding[0])
+            order = re.findall("[-+]?[.]?[\d]+(?:,\d\d\d)*[\.]?\d*(?:[eE][-+]?\d+)?", bonding[0])
+            print(order)
```

Comment:
```suggestion
            order = re.findall("[-+]?[.]?[\d]+(?:,\d\d\d)*[\.]?\d*(?:[eE][-+]?\d+)?", bonding[0])
```
Cool, this got more complicated. At least needs a comment, but we should probably finish the discussion on #1 about this first.